kernelci/runtime/docker.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, *args, **kwargs):
     def _load_env(self):
         if self.config.env_file and os.path.isfile(self.config.env_file):
             with open(self.config.env_file, encoding='utf-8') as env:
-                return [line.strip() for line in env.readlines()]
+                # skip # and empty lines
```

Comment:
Just a minor suggestion.
```suggestion
                # Skip comments and empty lines
```